### DISCUSSION TITLE
备份分区表时出现报错：Do not support backup table with temp partitions

### DISCUSSION THREAD

**Discussion by zhanghuajieHIT** [apache/doris#13451] [A - General / Q&A]
需要将一个集群的分区表搬迁到另一个集群，采用备份恢复的方式，但是当表中存在临时分区时报如下错误：
```
Do not support backup table with temp partitions
```

因此有如下疑问：
1、如果表中含有临时分区，可以采用什么高效的搬迁方式？
2、如果在备份的过程中，产生了临时分区，会对备份的数据或者原数据有影响吗？

**Comment by hf200012** [comment]
这个要先删除临时分区

**Reply by zhanghuajieHIT** [reply]
那么，如果在备份的过程中，产生了临时分区，会对备份的数据或者原数据有影响吗？

**Reply by zhanghuajieHIT** [reply]
临时分区是b，备份的是a分区，为什么会有影响呢？ @hf200012
